--- configure.orig	2020-04-12 21:10:50.798360997 -0500
+++ configure	2020-04-12 21:49:33.568232088 -0500
@@ -18516,15 +18516,7 @@
 	fi
 fi
 if test "x$cob_temp_flags" != "x"; then
-	cob_temp_flags=`echo "$cob_temp_flags" | sed -e 's/-g3//' -e 's/-g//' -e 's/ $//' -e 's/^ //'`
-	cob_temp_flags=`echo "$cob_temp_flags" | sed -e 's/[+-]O[0-9s]//' -e 's/ $//' -e 's/^ //'`
-	cob_temp_flags=`echo "$cob_temp_flags" | sed -e 's/-O//' -e 's/ $//' -e 's/^ //'`
-	cob_temp_flags=`echo "$cob_temp_flags" | sed -e 's/-fmessage-length=0//'`
-	cob_temp_flags=`echo "$cob_temp_flags" | sed -e 's/-D_FORTIFY_SOURCE=.//'`
-	cob_temp_flags=`echo "$cob_temp_flags" | sed -e 's/-fstack-protector//'`
-	cob_temp_flags=`echo "$cob_temp_flags" | sed -e 's/-funwind-tables//'`
-	cob_temp_flags=`echo "$cob_temp_flags" | sed -e 's/-fasynchronous-unwind-tables//'`
-	cob_temp_flags=`echo "$cob_temp_flags" | sed -e 's/  */ /g' -e 's/ $//' -e 's/^ //'`
+	cob_temp_flags=`echo "$cob_temp_flags" | sed -e 's/-fstack-protector-strong//' -e 's/-funwind-tables//' -e 's/-fasynchronous-unwind-tables//' -e 's/  */ /g' -e 's/ $//' -e 's/^ //' -e 's/-g3//' -e 's/ $//' -e 's/^ //'`
 fi
 
 if test "x$cob_temp_flags" != "x"; then
